### DISCUSSION TITLE
Make a 'mini-teleport' console command, not counting as cheats

### DISCUSSION THREAD

**Discussion by Sevenecho** [FakeFishGames/Barotrauma#8438] [Ideas and suggestions]
Me and my friends had to wipe 3 hours of progress because our Typhon-2 submarine stuck inside the terrain texture during last mission (artefact retrieval). Somehow it parked so deep into the ground it refused to go anywhere by an inch. We were 300 meters from our destination station, completely helpless. We tried to destroy the hull, to bump a sub with our bodies and explosions, we tried everything. Apparently, teleportation command could have saved us, but that would mean a 'cheats' flag-mark in our save file, with no further achievements.

I can understand TP is a cheat, but we had no intention getting from Abyss to an exit in one blink. All we needed is getting our sub unstuck, and 10 meters would have suffice. Can it really hurt the game so much to let players TP_[direction] their sub _once_ per round for a tiny bit? I can't imagine the possible exploits it might cause.

**Comment by AFK-TankyMOFO** [comment]
You mean like the unstuck function in Medieval Dynasty?

**Comment by Sevenecho** [comment]
Can't say for sure, never tried it.
Unstuck function like in Warframe would do a trick, but that would mean the game remembers your every step and can send you back for a while. I suppose a mini-teleport is easier to implement.
